test/E2ETests/project.json

```diff
@@ -21,6 +21,14 @@
                 "System.Net.Http": "",
                 "System.Xml": ""
             }
+        },
+        "dnxcore50": {
```

Comment:
Is there a way to disable running the coreclr version of the library on teamcity? I think the build.cmd runs once against all the targets configured.